server/GameOn/GameOn.Tournaments/TournamentsService.cs

```diff
@@ -60,11 +60,22 @@ internal async Task<ScoreResult[]> CalculatePlayerScores(string tenantId, string
                 throw new NotFoundException($"Tournament Id {tournamentId} is not found");
             var tournament = tournaments.First(t => t.Id == tournamentId);
 
+            // 1.4 Players must exist in Tournament
```

Comment:
Added a guard here as I was able to submit player Ids that did not exist in the Tournament 👍